### PR TITLE
UI Redesign

### PR DESCRIPTION
**Pull request by libauror-a**
funny thingy, i like redesigning uis

also code cleanup

~~maybe a commandline mode~~ ill do that in another pr

- [x] logo
- [x] use checkboxes
- [x] sad guy when it fails :((
- [x] fix file selector FST bug
- [ ] ~~new buttons~~

### CODE REVIEW

File: Project.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 	<!-- _________________________ Application Settings _________________________ -->
 
-	<app title="FLP Trial State modder. Perfect for people who don't buy FL Studio!" file="FLPTrialStateModder" main="Main" version="0.0.1" company="RafPlayz69YT" />
+	<app title="FLTrial" file="FLTrial" main="Main" version="0.0.1" company="RafPlayz69YT" />
```
Comment (raf13lol) [inline Project.xml]: you should change FLTrial to FLPTSM or something lol

File: source/Main.hx [hunk 2]
```diff
@@ -154,11 +168,13 @@ class PlayState extends FlxState
 			} // check it aint broken
 			var flp = sys.io.File.getBytes(path); // yoink the bytes
 
+			// hopefully flp.length is the same as flp.b.length
```
Comment (raf13lol) [inline source/Main.hx]: they are always

File: source/Main.hx [hunk 3]
```diff
@@ -98,41 +124,29 @@ class PlayState extends FlxState
...
+		trial.y += offsetY;
+
+		overwriteButton.scale.scale(1.2, 1.2);
+		// overwriteButton.updateHitbox(); why the hell does this fuck up the checkbox position
+		overwriteButton.textX += 15;
+		overwriteButton.y = untrialbutton.y - overwriteButton.height - 20;
+
+		add(overwriteButton);
+
+		FlxG.sound.play("assets/sounds/startup.wav"); // play that ding sound
```
Comment (raf13lol) [inline source/Main.hx]: ~~the start up wav isnt meant for start up (i kinda just copy and pasted the wav from something else of mine)~~
nvm read below

File: source/Main.hx [hunk 4]
```diff
@@ -16,16 +16,21 @@ import haxe.Int64;
 import haxe.io.Bytes;
 import haxe.io.BytesData;
 import haxe.io.Path;
+import hl.UI;
```
Comment (raf13lol) [inline source/Main.hx]: cant use in cpp

File: source/Main.hx (hunk 3 above)
Comment (raf13lol) [reply source/Main.hx]: wait nvm im dumb lolollolololo didnt realize that you made a new one

**Comment by raf13lol** [comment]
button text is small lol
image

**Comment by libauror-a** [comment]
yeah i have to fixies!!!

**Comment by raf13lol** [comment]
cool also idk if i should or you but i think there needs to be a new button for fst untrialing (file filter isnt working right today so yea)

**Comment by libauror-a** [comment]
> cool also idk if i should or you but i think there needs to be a new button for fst untrialing (file filter isnt working right today so yea)

hmmm.. let me try

**Comment by libauror-a** [comment]
bro what the hell???

i built with windows and this happened??? 
image

but when i built with hl, this appears??
image

also, wierd thing, the windows builds of any lime app have less DPI or less PPI

**Comment by raf13lol** [comment]
i dont use hl lol

**Comment by raf13lol** [comment]
idk why its chinese for you lol it works fine on compiled windows for me

**Comment by libauror-a** [comment]
i dont know, the only character i know from that random string is Ś

**Comment by libauror-a** [comment]
https://user-images.githubusercontent.com/75984453/211143140-71900b0c-9de5-4b75-abd4-b8a40b136c53.mp4

waiting for merge

**Comment by raf13lol** [comment]
FIX YOUR SPELLING GRAH!!!!
image

**Comment by libauror-a** [comment]
oh

**Comment by libauror-a** [comment]
i f'ianex fellinfg eraur

**Comment by raf13lol** [comment]
cool

**Comment by raf13lol** [comment]
what social do you want for credit

**Comment by libauror-a** [comment]
tag me in the readme and the changelog file, is that ok?

**Comment by raf13lol** [comment]
yea im asking what social you want credited

**Comment by libauror-a** [comment]
just [my github profile](https://github.com/ThePlank)

**Comment by raf13lol** [comment]
alright let me go test and ill merge

File: source/FunnyButton.hx [hunk 5]
```diff
@@ -18,7 +18,9 @@ class FunnyButton extends FlxButton
 {
 	public var state:ButtonState = NORMAL;
 
-	public function new(x:Int, y:Int, text:String, scale:FlxPoint, ?onPress:Void->Void)
+	public var enabled:Bool = true;
+
+	function new(x:Int, y:Int, text:String, scale:FlxPoint, ?onPress:Void->Void)
```
Comment (raf13lol) [inline source/FunnyButton.hx]: needs to be public for cpp :grin:

File: Project.xml [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 	<!-- _________________________ Application Settings _________________________ -->
 
-	<app title="FLTrial" file="FLTrial" main="Main" version="0.0.1" company="RafPlayz69YT" />
+	<app title="FLTSM" file="FLTSM" main="Main" version="0.0.1" company="RafPlayz69YT" />
```
Comment (raf13lol) [inline Project.xml:6]: maybe new acronymn should be flftsm (fruity loops file trial state modder)

Comment (raf13lol) [reply Project.xml:6]: you dont need to tho lol

Comment (libauror-a) [reply Project.xml:6]: ok i guess

Comment (libauror-a) [reply Project.xml:6]: domn't wanna remake the logo tho :((

Comment (raf13lol) [reply Project.xml:6]: i mean you dont have to

**Comment by raf13lol** [comment]
credits done and done

**Pull request merged**